Trakt/ServerMediator.cs

```diff
@@ -405,6 +412,7 @@ private async void KernelPlaybackProgress(object sender, PlaybackProgressEventAr
 
                             _playbackState[traktUser.LinkedMbUserId].PlaybackProgress = playbackProgressEventArgs.PlaybackPositionTicks ?? 0L;
                             _playbackState[traktUser.LinkedMbUserId].IsPaused = false;
```

Comment:
Is setting `IsPaused = false` always appropriate here?

Reply:
See new commit.